Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.67-slim AS builder
```

Comment:
You meant to put this in implementations/rust-boon

Reply:
I just copied it from rust-jsonschema/Dockerfile which uses the same.
do you suggest a change?

Reply:
Your PR is adding a file called `Dockerfile` to the root of this repository, rather than to your implementation directory.

Reply:
it seems your forced push has messed with my git and I got confused in resolving conflict. let me recheck